cassandra/src/main/scala/org/apache/zeppelin/cassandra/DisplaySystem.scala

```diff
@@ -507,11 +558,15 @@ object MetaDataConverter {
         .map(c => new ColumnDetails(c.columnMeta.getName, StaticColumn, c.columnMeta.getType))
     }
 
-    protected def convertClusteringColumns(columns: List[ColumnMetaWrapper], orders: List[DriverClusteringOrder]): List[ColumnDetails] = {
+    protected def convertClusteringColumns(
+      columns: List[ColumnMetaWrapper],
+      orders: List[DriverClusteringOrder]
+    ): List[ColumnDetails] = {
```

Comment:
When single line length greater than 100 or 120, I followed spark style :) 
This style is not ten perfect, i prefer below two: 1. two space indent of paramter; 2. more clearly for muti-paramter function list one by one line

> java max line length is 100
> scala max line length.(If 100, there too much lines to modify)

change to default style?

```
protected def convertClusteringColumns(columns: List[ColumnMetaWrapper], 
								 orders: List[DriverClusteringOrder]): List[ColumnDetails] = {
										
```